docs/en/04_Changelogs/4.9.0.md

```diff
@@ -1,6 +1,40 @@
 # 4.9.0 (Unreleased)
 
+## Overview
 
-## New features
+- [Features and enhancements](#features-and-enhancements)
+    - [Image lazy loading](#image-lazy-loading)
+    - [Other new features](#other-features)
+- [Bugfixes](#bugfixes)
+
+
+## Features and enhancements {#features-and-enhancements}
+
+### Image lazy loading {#image-lazy-loading}
+
+Most images will now get the loading="lazy" attribute added to them.  This includes images added via the WYSIWYG insert 
+media button as well as image DataObjects added to templates.  Image HTML tags (`&lt;img&gt;`) added in templates will not
+be lazy loaded by default
```

Comment:
I'm pretty sure MD is smart enough to know that `<` and `>` symbols in code blocks should be escaped. Otherwise we would have broken bits of HTML throughout our doc.

```suggestion
Most images will now get the `loading="lazy"` attribute added to them.  This includes:
- images added via the HTML Editor's ***insert media*** button
- image DataObjects added to templates.  

`<img>` HTML tags added in templates will not be lazy loaded by default.
```

Reply:
Updated